package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "stackblitz-starters",
+  "private": true,
+  "scripts": {
+    "format": "prettier --write '**/*'",
+    "format:check": "prettier --check '**/*'"
```

Comment:
Passing all files to Prettier. We will then use the `.prettierignore` to exclude files that should not be formatted.

Alternatively we could use a pattern such as `'**/*.{html,css,scss,json,js,jsx,ts,tsx}'`, but:

- that doesn't include files such as `.stackblitzrc`;
- it would be easy to miss files that way;
- I'd rather have Prettier complain that a file is not in a format that it recognizes, so that we can know that we need to tweak our `.prettierrc` or add that file type to `.prettierignore`.